domain/src/test/kotlin/camp/nextstep/edu/github/domain/UseCaseTest.kt

```diff
@@ -0,0 +1,39 @@
+package camp.nextstep.edu.github.domain
+
+import camp.nextstep.edu.github.domain.model.GithubRepositoryInfo
+import camp.nextstep.edu.github.domain.repository.GithubRepository
+import camp.nextstep.edu.github.domain.usecase.GetGithubRepositoriesUseCase
+import com.google.common.truth.Truth.assertThat
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+@ExperimentalCoroutinesApi
+class UseCaseTest {
```

Comment:
```suggestion
class GetGithubRepositoriesUseCaseTest {
```
테스트 파일을 만들 때에는 `클래스명Test` 로 작성하는 것이 좋아요.